.github/merge_rules.json

```diff
@@ -41,6 +41,22 @@
       "approved_by": ["linbinyu", "kit1980", "IvanKobzarev", "dreiss"],
       "mandatory_checks_name": ["Facebook CLA Check", "Lint"]
    },
+   {
+      "name": "FFT",
+      "patterns": [
+         "aten/src/ATen/native/cuda/*FFT*.h",
+         "docs/source/fft.rst",
+         "torch/fft/**",
+         "torch/include/**/*fft*.h",
+         "torch/include/ATen/native/cuda/*FFT*.h",
```

Comment:
`torch/include` isn't actually committed to git, it's generated during the build process.

```suggestion
```

Reply:
oooooh haha thanks! good to know :)